src/promptflow/promptflow/_core/tool_meta_generator.py

```diff
@@ -376,13 +376,27 @@ def generate_tool_meta_dict_by_file(path: str, tool_type: ToolType):
 def generate_tool_meta(
     working_dir: Path,
     tools: Mapping[str, Mapping[str, str]],
-    tool_dict: dict,
-    exception_dict: dict,
     prevent_terminate_signal_propagation: bool = False,
 ):
-    # This method might run in a child process, and when executed within a uvicorn app,
```

Comment:
Please keep this comments

Reply:
Or move this logic to _generate_tool_meta_and_update_dict

Reply:
it has been moved to docstring

Reply:
need more information to determine if we should move it to `_generate_tool_meta_and_update_dict`